### PR TITLE
Add a script to update Network Operator dependencies

### PR DESCRIPTION
**Pull request by e0ne**

### CODE REVIEW

File: scripts/update-deps.sh
Comment (rollandf) [inline scripts/update-deps.sh:19]: I think this is needed, no?

File: scripts/update-deps.sh
Comment (rollandf) [inline scripts/update-deps.sh:20]: Does the `$this` works?

File: scripts/update-deps.sh
Comment (rollandf) [inline scripts/update-deps.sh:22]: Maybe add indentation the same as the others

File: scripts/update-deps.sh
Comment (rollandf) [inline scripts/update-deps.sh:23]: Typo, should be `--mofed`

**Review by rollandf — CHANGES_REQUESTED** [review]

File: scripts/update-deps.sh
Comment (e0ne) [reply scripts/update-deps.sh:19]: fixed

File: scripts/update-deps.sh
Comment (e0ne) [reply scripts/update-deps.sh:20]: fixed

File: scripts/update-deps.sh
Comment (e0ne) [reply scripts/update-deps.sh:22]: Such indentation is needed to be shown in CLI correctly

File: scripts/update-deps.sh
Comment (e0ne) [reply scripts/update-deps.sh:23]: thanks, fixed

File: scripts/update-deps.sh
Comment (adrianchiris) [inline scripts/update-deps.sh:91]: do you need the while loop ?
you can just use case statement as you are not shifting and exiting in every switch

perhaps check that $# equals 2 or fail otherwise first.

unless you want to support update of multiple components in the same call, then you need to modify this shell script to support it.

File: scripts/update-deps.sh
Comment (adrianchiris) [inline scripts/update-deps.sh:88]: nit: too many empty lines

**Comment by adrianchiris** [comment]
@e0ne  once comments addressed can be merged IMO

@rollandf please give it an LGTM once you are happy

File: scripts/update-deps.sh
Comment (adrianchiris) [inline scripts/update-deps.sh:31]: @e0ne could you add also an example section with a run example just for clarity

File: scripts/update-deps.sh
Comment (adrianchiris) [inline scripts/update-deps.sh:39]: we could leverage kustomize to do the grunt work

**Comment by e0ne** [comment]
Will be implemented in #450

**Pull request closed without merging**
